src/models/sensor-gdx-manager.ts

```diff
@@ -0,0 +1,220 @@
+import { SensorConfiguration } from "./sensor-configuration";
+import { SensorManager, NewSensorData } from "./sensor-manager";
+import { SensorConfig } from "@concord-consortium/sensor-connector-interface";
+import godirect from "@vernier/godirect"
+
+export class SensorGDXManager extends SensorManager {
+    supportsDualCollection = true;
+
+    private internalConfig: SensorConfig;
+    private hasData: boolean = false;
+    private stopRequested: boolean = false;
+    private disconnectRequested: boolean = false;
+    private gdxDevice: any;
+    private enabledSensors: any;
+    private initialColumnNum = 100;
+
+    constructor() {
+      super();
+      // create SensorConfiguration
+      // This should be improved, we don't need all of these properties when making
+      // a new sensor manager. The SensorConfiguration class could be have an
+      // interface so then sensor managers can provide their own implementation
+      this.internalConfig = {
+        collection:{ canControl:true, isCollecting:false },
+        columnListTimeStamp: new Date(),
+        columns:{ },
+        currentInterface: "Vernier Go Direct",
+        currentState: "unknown",
+        os: { name: "Fake", version: "1.0.0"},
+        requestTimeStamp: new Date(),
+        server: { arch: "Fake", version: "1.0.0" },
+        sessionDesc: "Fake",
+        sessionID: "1234",
+        sets:{
+          "100": {
+            name: "Run 1",
+            colIDs: [100]
+          }
+        }
+      };
+    }
```

Comment:
Any thoughts on these initial `internalConfig` values?  Much of this is based on the sensor tag implementation, but it seems like we should remove some of these values that serve no purpose or are potentially misleading.